### PR TITLE
[ET-808] Add "Not Going" option to classic editor if using new RSVP views

### PR DESCRIPTION
**Pull request by sc0ttkclark**
[ET-808]

Screencast: https://share.skc.dev/4gujp87Q

[ET-808]: https://moderntribe.atlassian.net/browse/ET-808

### CODE REVIEW

File: src/Tribe/RSVP.php [hunk 1]
```diff
@@ -118,6 +118,12 @@ class Tribe__Tickets__RSVP extends Tribe__Tickets__Tickets {
 	 */
 	public $deleted_product = '_tribe_deleted_product_name';
 
+	/**
+	 * Meta key that holds the "not going" option visibility status.
+	 * @var string
```
Comment (tr1b0t) [inline src/Tribe/RSVP.php:123]: :no_entry_sign: There must be exactly one blank line before the tags in a doc comment

<sub>hash: dd347afdf19e3a6741749462d984517e</sub>

File: src/admin-views/rsvp-metabox-not-going.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	class="input_block ticket_advanced_Tribe__Tickets__RSVP tribe-dependent"
+	data-depends="#Tribe__Tickets__RSVP_radio"
+	data-condition-is-checked
+>
+	<label
+		for="Tribe__Tickets__RSVP_not_going"
+		class="ticket_form_label ticket_form_left"
+	>
+		<?php esc_html_e( 'Not Going:', 'event-tickets' ); ?>
+	</label>
```
Comment (tr1b0t) [inline src/admin-views/rsvp-metabox-not-going.php:11]: :no_entry_sign: Missing file doc comment

<sub>hash: f1665f85d3e265c52f30c818501ab17f</sub>

File: src/admin-views/rsvp-metabox-not-going.php [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	<input
+		type='checkbox' id='Tribe__Tickets__RSVP_not_going'
+		name='tribe-ticket[not_going]'
+		class="ticket_field tribe-rsvp-field-not-going ticket_form_right"
+		value='yes'
```
Comment (juanfra) [inline src/admin-views/rsvp-metabox-not-going.php:16]:
```suggestion
		type="checkbox"
		id="Tribe__Tickets__RSVP_not_going"
		name="tribe-ticket[not_going]"
		class="ticket_field tribe-rsvp-field-not-going ticket_form_right"
		value="yes"
```

File: src/admin-views/rsvp-metabox-not-going.php [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<div
+	class="input_block ticket_advanced_Tribe__Tickets__RSVP tribe-dependent"
+	data-depends="#Tribe__Tickets__RSVP_radio"
+	data-condition-is-checked
+>
+	<label
+		for="Tribe__Tickets__RSVP_not_going"
```
Comment (paulmskim) [inline src/admin-views/rsvp-metabox-not-going.php:7]: not crucial, seeing as this is admin views, but better if we can keep this and the corresponding id field lowercase (the class `tribe-rsvp-field-not-going` would be a great id for this)

File: src/Tribe/RSVP.php (hunk 1 above)
Comment (sc0ttkclark) [reply src/Tribe/RSVP.php:123]:
```suggestion
	 *
	 * @var string
```

File: src/admin-views/rsvp-metabox-not-going.php (hunk 4 above)
Comment (sc0ttkclark) [reply src/admin-views/rsvp-metabox-not-going.php:7]: I copied from another admin view, just replaced a few things. Will update

File: src/admin-views/rsvp-metabox-not-going.php [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Admin setting for "Can't Go" checkbox when creating/editing RSVP in Classic Editor.
+ */
+?>
```
Comment (tr1b0t) [inline src/admin-views/rsvp-metabox-not-going.php:5]: :no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: adaab674d61193fb55e069f0a4c5e78f</sub>

Comment (sc0ttkclark) [inline src/admin-views/rsvp-metabox-not-going.php:5]:
```suggestion

?>
```

File: src/admin-views/rsvp-metabox-not-going.php [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+	<input
+		type="checkbox"
+		id="tribe-tickets-rsvp-not-going"
+		name="tribe-ticket[not_going]"
+		class="ticket_field tribe-rsvp-field-not-going ticket_form_right"
+		value="yes"
+		<?php checked( $not_going ); ?>
+	/>
+	<span class="tribe_soft_note ticket_form_right"><?php esc_html_e( "Enable \"Can't Go\" responses", 'event-tickets' ); ?></span>
+</div>
```
Comment (sc0ttkclark) [inline src/admin-views/rsvp-metabox-not-going.php:28]:
```suggestion
</div>

```

**Pull request merged**